src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -236,19 +236,6 @@ internal static IntPtr RhHandleAllocDependent(object primary, object secondary)
             return h;
         }
 
-        // Allocate variable handle with its initial type.
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        [RuntimeImport(RuntimeLibrary, "RhpHandleAllocVariable")]
-        private static extern IntPtr RhpHandleAllocVariable(object value, uint type);
```

Comment:
This may be also needed for ComWrappers lifetime tracking eventually, but it is fine to delete it for now.